packages/teams-js/src/private/externalAppAuthentication.ts

```diff
@@ -267,14 +315,14 @@ export namespace externalAppAuthentication {
       [
         appId,
         originalRequestInfo,
-        authenticateParameters.url,
+        authenticateParameters.url.href,
```

Comment:
What's the purpose of being href? Is it to make it unchangeable?

Reply:
It's the equivalent of `toString()`. we don't want to modify it, so we can use either (but yeah if we wanted to modify the value we would have to use `href`